x-pack/plugins/actions/server/config.ts

```diff
@@ -72,6 +72,8 @@ const connectorTypeSchema = schema.object({
   maxAttempts: schema.maybe(schema.number({ min: MIN_MAX_ATTEMPTS, max: MAX_MAX_ATTEMPTS })),
 });
 
+export const DEFAULT_USAGE_API_URL = 'https://usage-api.elastic-system/api/v1/usage';
```

Comment:
We can't do this, as this way of hard-coding the URL broke, when the URL changed :-)

We're in the process of figuring out what to do with these, but we likely need a central location (core? new plugin?) to maintain these things, including the ca certs that we've already added.